packages/react-instantsearch-dom/src/lib/voiceSearchHelper/__tests__/index.ts

```diff
@@ -0,0 +1,172 @@
+import VoiceSearchHelper, { VoiceSearchHelperParams } from '..';
+
+// copied from InstantSearch.js
```

Comment:
You can add a [permanent link](https://help.github.com/en/articles/getting-permanent-links-to-files) to the original file.